flamedisx/lxe_blocks/detection.py

```diff
@@ -123,13 +133,17 @@ def _compute(self, data_tensor, ptensor,
 class DetectElectrons(DetectPhotonsOrElectrons):
     dimensions = ('electrons_produced', 'electrons_detected')
 
-    special_model_functions = ('electron_acceptance',)
+    special_model_functions = ('electron_acceptance', 'extraction_eff')
     model_functions = ('electron_detection_eff',) + special_model_functions
 
     @staticmethod
     def electron_detection_eff(drift_time, *,
-                               elife=452e3, extraction_eff=0.96):
-        return extraction_eff * tf.exp(-drift_time / elife)
+                               elife=452e3):
+        return 1. * tf.exp(-drift_time / elife)
+
+    @staticmethod
+    def extraction_eff(nel):
```

Comment:
Could you make the extraction eff a keyword argument just like elife above / as it was before?

Reply:
Good point, fairly easy to add!

Reply:
Thanks!